src/dispatching/dispatcher.rs

```diff
@@ -280,6 +302,92 @@ where
     }
 }
 
+const WORKER_QUEUE_SIZE: usize = 64;
```

Comment:
This should probably be configureable in dispatcher?

Reply:
Anyone cares?

Reply:
Yes

Reply:
No